templates/aurora_postgres.template.yaml

```diff
@@ -118,6 +118,9 @@ Outputs:
   DBName: 
     Description: "Amazon Aurora database name"
     Value: !Ref DBName
+  DBMasterUsername:
+    Description: "Amazon Aurora database username"
```

Comment:
Please change "Amazon Aurora database username" to ""Amazon Aurora database master username"

Reply:
Fixed in  b46f9681159d19888be236184ee7bb00b6fd6752